keras_cv/layers/preprocessing/grayscale.py

```diff
@@ -13,19 +13,17 @@
 # limitations under the License.
 
 import tensorflow as tf
-from tensorflow.keras import backend
-from tensorflow.keras import layers
 
 
-class Grayscale(layers.Layer):
+class Grayscale(tf.keras.__internal__.layers.BaseImageAugmentationLayer):
     """Grayscale is a preprocessing layer that transforms RGB images to Grayscale images.
     Input images should have values in the range of [0, 255].
 
     Input shape:
-        3D (unbatched) or 4D (batched) tensor with shape:
+        4D (batched) tensor with shape:
```

Comment:
revert - can be either unbatched or batched

Reply:
done!